compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/CheckersConfiguration.kt

```diff
@@ -5,12 +5,14 @@
 
 package org.jetbrains.kotlin.fir.checkers.generator
 
+import org.jetbrains.kotlin.fir.FirElement
```

Comment:
Commit message: add one more checker type to Component Generator. "Add addition" is a tautology

Reply:
Also, you don't need description in commit message (line 3) in case when it matches commit message title (line 1).